new_client/src/auth/Signup.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Container, Paper, Grid, TextField, Button } from '@material-ui/core';
+import "./Error.css";
+
+export default function Signup() {
```

Comment:
The user schema has more fields than the ones required here.

Reply:
Resolved as mentioned by @AliabbasMerchant